.gitignore

```diff
@@ -4,6 +4,7 @@ htdocs/conf/conf.php.sav
 htdocs/conf/conf.php.back
 documents/
 custom2/
+htdocs/custom
```

Comment:
We want to keep the custom dir with its readme as part of code to be sure it exists when we install with a git clone. So it must not be excluded by gitignore. However you can include your module dir into the gitignore into custom.

Reply:
I fix it, thx for tour review